kernel/src/lib/nuumshell/shell.cr

```diff
@@ -0,0 +1,66 @@
+require "./apps/*"
+
+class NuumShell
+  def initialize
+    @user = "root"
+    @pass = "root"
+    @login = false
+    @max_login = 3
```

Comment:
What happens if the max login threshold is reached? Does the user get locked out from the kernel? Seems kinda suboptimal.